fitbenchmarking/fitbenchmark_one_problem.py

```diff
@@ -40,6 +40,19 @@
 
 def fitbm_one_problem(prob, minimizers, use_errors=True, group_results_dir=None):
     """
+    Sets up the workspace, cost function and function definitons for
+    a particular problem and fits the models provided in the problem
+    object. The best fit, along with the data and a starting guess
+    is then plotted on a visual display page.
```

Comment:
Does this function actually do this "The best fit, along with the data and a starting guess
    is then plotted on a visual display page."

Reply:
Yes, it calls make_plots which then plots the results.